ledger/streaming/client/benches/bench.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(test)]
+
+extern crate test;
+
+use futures::{executor::block_on, StreamExt};
+use mc_ledger_db::test_utils::MockLedger;
+use mc_ledger_streaming_api::{test_utils::get_custom_block_stream, BlockStream};
+use mc_ledger_streaming_client::block_validator::BlockValidator;
+use test::Bencher;
+
+/// Process 1000 typically sized blocks
+#[bench]
+fn process_1000_blocks(b: &mut Bencher) {
+    let logger = mc_common::logger::create_test_logger("benchmark_logger".to_string());
```

Comment:
Let's include the method name in the logger name.
```suggestion
    let logger = mc_common::logger::create_test_logger("benchmark:process_1000_blocks".to_string());
```

Reply:
Yeah agreed, contextual naming == better, I'll change it to show it's benching 1000 for the validator